b2share/modules/schemas/root_schemas/root_schema_v0.json

```diff
@@ -5,132 +5,191 @@
                 "type": "array",
                 "items": {
-                    "type": "string",
-                    "enum": ["Audiovisual", "Collection", "Dataset", "Event", "Image",
-                        "InteractiveResource", "Model", "PhysicalObject", "Service",
-                        "Software", "Sound", "Text", "Workflow", "Other"]
+                    "type": "object",
+                    "properties": {
+                        "resource_type": {
+                            "type": "string"
+                        },
+                        "resource_type_general": {
+                            "enum": ["Audiovisual", "Collection", "Dataset", "Event", "Image", "InteractiveResource", "Model", "PhysicalObject", "Service", "Software", "Sound", "Text", "Workflow", "Other"]
+                        }
+                    }
                 },
                 "uniqueItems": true
             },
-            "alternate_identifier": {
-                "title": "Alternate identifier.",
+            "alternate_identifiers": {
+                "title": "Alternate identifiers",
                 "description": "Any kind of other reference such as a URN, URI or an ISBN number.",
-                "type": "string"
+                "type": "array",
+                "items": {
+                    "type": "object",
+                    "properties": {
+                        "alternate_identifier": {
+                            "type": "string"
+                        },
+                        "alternate_identifier_type": {
+                            "type": "string"
+                        }
+                    }
+                },
+                "uniqueItems": true
+            },
+            "descriptions": {
+                "title": "Descriptions",
+                "description": "A more elaborate description of the resource. Focus on a content description that makes it easy for others to find, and to interpret its relevance.",
+                "type": "array",
+                "items": {
+                    "type": "object",
+                    "properties": {
+                        "description": {
+                            "type": "string"
+                        },
+                        "description_type": {
+                            "enum": ["Abstract", "Methods", "SeriesInformation", "TableOfContents", "TechnicalInfo", "Other"]
+                        }
+                    }
+                },
+                "uniqueItems": true
             },
             "version": {
                 "title": "Version",
                 "description": "Denote the version of the resource.",
                 "type": "string"
             },
-            "publisher": {
-                "title": "Publisher",
-                "description": "The entity responsible for making the resource available, either a person, an organization, or a service.",
+            "contact_email": {
```

Comment:
Note for later: This should not be indexed in B2Find. At some point we will probably remove the email address and just provide a contact form in the UI.